src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -272,14 +272,15 @@ public static List<HearingLocation> getHearingLocations(SscsCaseData caseData,
             .getEpimsIdForVenue(caseData.getProcessingVenue())
             .orElse(null);
 
-        HearingLocation hearingLocation = new HearingLocation();
-        hearingLocation.setLocationId(epimsId);
-        hearingLocation.setLocationType(COURT);
-
-        List<HearingLocation> hearingLocations = new ArrayList<>();
-        hearingLocations.add(hearingLocation);
+        Resource resource = new ClassPathResource("multipleHearingLocations.json");
+        ConcurrentHashMap<String,List<String>> multipleHearingLocations = new ObjectMapper().readValue(Files.newInputStream(Paths.get(resource.getURI())), new TypeReference<ConcurrentHashMap<String, List<String>>>(){});
 
-        return hearingLocations;
+        return multipleHearingLocations.values().stream()
+            .filter(listValues ->  listValues.contains(epimsId))
+            .findFirst()
+            .orElseGet(() -> Collections.singletonList(epimsId))
+            .stream().map(epims -> HearingLocation.builder().locationId(epims).locationType(COURT).build())
+            .collect(Collectors.toCollection(ArrayList::new));
```

Comment:
(Minor) could use collections.toList

Reply:
When I prepared the snipped I chose this more explicit way of writing it as I noticed that the method was already there and was returning an ArrayList. Not knowing if there was a specific reason to return ArrayList, I decided to play it safe.

I would keep it as is, also keeping in mind for new methods that we should definitely do what you recommend unless there's a specific reason to use another list implementation.